docs/container/drivers/joyent.rst

```diff
@@ -17,7 +17,7 @@ Instantiating the driver
 Download the script::
 
     curl -O https://raw.githubusercontent.com/joyent/sdc-docker/master/tools/sdc-docker-setup.sh
-    
+
```

Comment:
if you wouldn't mind just undoing the changes to the joyent file, `git checkout trunk docs/container/drivers/joyent.rst`